app/models/registration_wizard.rb

```diff
@@ -150,7 +150,7 @@ def answers
     end
 
     array << OpenStruct.new(key: "Course",
-                            value: query_store.course.name,
+                            value: I18n.t(query_store.course.identifier, scope: "helpers.label.registration_wizard.course_identifier_options"),
```

Comment:
Makes sure that the option the user sees on the CYA page matches the option they selected.